docs/sources/flow/reference/components/otelcol.receiver.awsfirehose.md

```diff
@@ -0,0 +1,163 @@
+---
+aliases:
+- /docs/grafana-cloud/agent/flow/reference/components/otelcol.receiver.awsfirehose/
+- /docs/grafana-cloud/monitor-infrastructure/agent/flow/reference/components/otelcol.receiver.awsfirehose/
+- /docs/grafana-cloud/monitor-infrastructure/integrations/agent/flow/reference/components/otelcol.receiver.awsfirehose/
+- /docs/grafana-cloud/send-data/agent/flow/reference/components/otelcol.receiver.awsfirehose/
+canonical: https://grafana.com/docs/agent/latest/flow/reference/components/otelcol.receiver.awsfirehose/
+description: Learn about otelcol.receiver.awsfirehose
+title: otelcol.receiver.awsfirehose
+---
+
+# otelcol.receiver.awsfirehose
+
+`otelcol.receiver.awsfirehose` accepts cloudwatch json-formatted traces over the network and
+forwards it to other `otelcol.*` components.
```

Comment:
The [upstream](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/awsfirehosereceiver) component mentions this as its intro:

```
Receiver for ingesting AWS Kinesis Data Firehose delivery stream messages and parsing the records received based on the configured record type.
```

Are the two statements compatible (I'm not familiar with firehose tbh). Also, just to make sure, does this receiver receive JSON-formatted traces and forwards metrics generated out of them?

Finally, the upstream docs mention that:
```
The AWS Kinesis Data Firehose Delivery Streams currently only support HTTPS endpoints using port 443. This can be potentially circumvented using a Load Balancer.
```

Is it applicable to this component as well? Should we copy this over?

Reply:
To answer your first question, yes, the two statements are compatible, as AWS Kinesis Data Firehose Delivery Streams is the service used by Cloudwatch to send metrics  streams to an HTTP endpoint. I will clarify it in the doc.

For the second point, I did not explain it clearly. The receiver receives JSON-formatted metrics from AWS and forward them to other otel components.

As for the HTTPS-only endpoint, since it's a requirement from the AWS side, I will add the information to the doc.